pkg/api/validation/validation.go

```diff
@@ -133,12 +133,25 @@ func ValidatePodSpecificAnnotations(annotations map[string]string, spec *api.Pod
 	allErrs = append(allErrs, ValidateSeccompPodAnnotations(annotations, fldPath)...)
 	allErrs = append(allErrs, ValidateAppArmorPodAnnotations(annotations, spec, fldPath)...)
 
-	sysctls, err := api.SysctlsFromPodAnnotation(annotations[api.SysctlsPodAnnotationKey])
+	sysctlAnn := annotations[api.SysctlsPodAnnotationKey]
+	sysctls, err := api.SysctlsFromPodAnnotation(sysctlAnn)
 	if err != nil {
-		allErrs = append(allErrs, field.Invalid(fldPath.Key(api.SysctlsPodAnnotationKey), annotations[api.SysctlsPodAnnotationKey], err.Error()))
+		allErrs = append(allErrs, field.Invalid(fldPath.Key(api.SysctlsPodAnnotationKey), sysctlAnn, err.Error()))
 	} else {
 		allErrs = append(allErrs, validateSysctls(sysctls, fldPath.Key(api.SysctlsPodAnnotationKey))...)
 	}
+	unsafeSysctlAnn := annotations[api.UnsafeSysctlsPodAnnotationKey]
+	unsafeSysctls, err := api.SysctlsFromPodAnnotation(unsafeSysctlAnn)
+	if err != nil {
+		allErrs = append(allErrs, field.Invalid(fldPath.Key(api.UnsafeSysctlsPodAnnotationKey), unsafeSysctlAnn, err.Error()))
+	} else {
+		allErrs = append(allErrs, validateSysctls(sysctls, fldPath.Key(api.UnsafeSysctlsPodAnnotationKey))...)
+	}
+	inBoth := sysctlIntersection(sysctls, unsafeSysctls)
+	for i := range inBoth {
```

Comment:
As it's the error case it's not performance relevant. I would leave it as it is.